it/v3/endpoints/CreateAndAmendEmploymentExpensesControllerISpec.scala

```diff
@@ -45,8 +44,7 @@ class CreateAndAmendEmploymentExpensesControllerISpec extends IntegrationBaseSpe
           DownstreamStub.onSuccess(DownstreamStub.PUT, downstreamUri, NO_CONTENT, JsObject.empty)
 
         val response: WSResponse = await(request().put(requestBodyJson))
-        response.status shouldBe OK
-        response.json shouldBe hateoasResponse
```

Comment:
this can be replaced to verify that there is no body
response.body.isEmpty shouldBe true